Alias Manager for Configurations/AliasManagerForConfigurations.FCMacro

```diff
@@ -0,0 +1,355 @@
+
+# ============================================================================================================
+# ============================================================================================================
+# ==                                                                                                        ==
+# ==                                           Alias Manager for Configurations                             ==
+# ==                                                                                                        ==
+# ============================================================================================================
+# ============================================================================================================
+# ABOUT
+# ============================================================================================================
+# version v0.10
+# Macro developed for FreeCAD (http://www.freecadweb.org/).
+# This macro helps managing aliases inside FreeCAD Spreadsheet workbench. It is a variant of alias Manager by 
+# 2016 tarihatari & Pablo Gil Fernandez and EasyAlias by TheMarkster and rosta
+# It is able to:
+#        -set aliase of a cell in a horizontal table, e.g. Configuration Table, or vertical tabel in an older 
+#         scheme know as a Part Family.  You select the cell or cells and hit Execute button.  Based on the
+#         radio button selected the alias label is taken from above the cells or to the left of the cells.
+#       - there is also a clear fucntion which removes the alias in selected cells if that helps too.
+#       - the dialog used is modal and stays on top.  This way a user can have open while working in the
+#         and it's always available to add to adjust aliases.  It can be moved off to the side or to a second
+#         display.  It can also be minimized and brought up when needed.
+#
+# This has ONLY be tested on Mac OS.
+#
+# LICENSE
+# ============================================================================================================
+# Original work done by tarihatari (https://github.com/tarihatari/FreeCAD_Macros) &
+# done by TheMarkster and rosta (https://wiki.freecadweb.org/Macro_EasyAlias)
+# Improved by Pablo Gil Fernandez
+# Improved by Ron Zancola
+#
+# Copyright (c) 2023 tarihatari & TheMarkster and rosta & Pablo Gil Fernandez & Ron Zancola
+#
+# This work is licensed under GNU Lesser General Public License (LGPL).
+# To view a copy of this license, visit https://www.gnu.org/licenses/lgpl-3.0.html.
+#
+# ============================================================================================================
+__title__   = "Alias Manager for Configuration Tables"
```

Comment:
Please use [the correct metadata format](https://github.com/FreeCAD/FreeCAD-macros/#macro-metadata).